client/src/lib/hooks/useInputValidator.tsx

```diff
@@ -1,16 +1,21 @@
-import { useState } from 'react';
+import { useState, useCallback, DependencyList } from 'react';
 
 function useInputValidator(
   initialValue: string,
   validator: (user_input: string) => string,
+  deps: DependencyList = [],
 ) {
   const [input, setInput] = useState(initialValue);
-  const [warning, setWarning] = useState('');
+  const [warning, setWarning] = useState(' ');
 
-  const handleInput = (user_input: string) => {
+  const onInput = (user_input: string) => {
```

Comment:
인자에 underscore가 들어가네요..!
```suggestion
  const onInput = (userInput: string) => {
```

Reply:
제가 잠시 파이썬세상에 갔다왔습니다.
수정할게요 ㅋㅋㅋ